frontend/device/input.lua

```diff
@@ -176,6 +176,14 @@ local Input = {
         },
     },
 
+    fake_event_set = {
+        IntoSS = true, OutOfSS = true,
+        UsbPlugIn = true, UsbPlugOut = true,
+        Charging = true, NotCharging = true,
+        WakeupFromSuspend = true, ReadyToSuspend = true,
+        UsbDevicePlugIn = true, UsbDevicePlugOut = true,
+    },
```

Comment:
Extracted the event names into a set when I needed to add one more check if event is fake.